src/FS.Globbing/baseline.netcore.json

```diff
@@ -473,1179 +473,6 @@
       ],
       "GenericParameters": []
     },
-    {
```

Comment:
Why are there 1300 line deletions here?

Reply:
Universe sets ExcludeInternalsPublic to true in order to only compare public items, but that parameter also controls how the files are generated. since these baselines a copied from the 2.1 Universe build the pub-ternals got removed. I filed aspnet/BuildTools#693 about the issue. I can attempt to modify my script to generate these from the 2.1.1 tag instead of from Universe, or we can deal with it for this time and make sure aspnet/BuildTools#693 is done for next release. Which do you prefer?

Reply:
I'm neutral on this one. Just wanted to be sure we knew why it was happening.